lib/lambda_ethereum_consensus/validator/duties.ex

```diff
@@ -27,26 +26,38 @@ defmodule LambdaEthereumConsensus.Validator.Duties do
 
   @type proposer_duty :: Types.validator_index()
 
+  @type sync_committee_aggregator_duty :: %{
+          aggregated?: boolean(),
+          selection_proof: Bls.signature(),
+          contribution_domain: Types.domain(),
+          subcommittee_index: Types.uint64()
+        }
+
   @type sync_committee_duty :: %{
-          # Given that we send messages in EVERY slot, instead
-          # of tracking them all, we just keep track of the last
-          # slot we broadcasted to avoid double publishs.
-          last_slot_broadcasted: Types.slot(),
+          broadcasted?: boolean(),
+          message_domain: Types.domain(),
+          validator_index: Types.validator_index(),
           subnet_ids: [Types.uint64()],
-          validator_index: Types.validator_index()
+          aggregation: [sync_committee_aggregator_duty()]
```

Comment:
Duties haven't been refactored, if we agreed with this way to represent them we might want to move attestation duties to follow this same pattern, separating the main attestation from its aggregation duty. This shouldn't impact the code a lot but should be easier to read and clearly separate the two kind of duties.